### PR TITLE
Fixed blog examples

### PR DESCRIPTION
**Pull request by andreujuanc**
### Proposed changes
- Removed invalid symbols
- Typescript Compatible
- Missing import statement
- Changed resources variable references to fix type mismatch between password and host

### CODE REVIEW

File: content/blog/managing-your-mysql-databases-with-pulumi/index.md [hunk 1]
```diff
@@ -55,10 +55,12 @@ In this example, we will create a database in the previously configured MySQL se
 
 ```javascript
 import * as mysql from '@pulumi/mysql';
-import * as pulumi from ‘@pulumi/pulumi’;
+import * as pulumi from '@pulumi/pulumi';
 
 const config = new pulumi.Config();
-const jdoePassword = config.requireSecret(‘jdoePassword’);
+const mysqlConfig = new pulumi.Config();
```
Comment (leezen) [inline content/blog/managing-your-mysql-databases-with-pulumi/index.md:61]: It's a bit confusing now to have both `config` and `mysqlConfig`

**Comment by leezen** [comment]
@andreujuanc Really appreciate your original help here. Any interest in updating this PR?

**Comment by andreujuanc** [comment]
Hi @leezen  , Sure, I just got my agenda upside-down ATM.

**Comment by leezen** [comment]
@andreujuanc Just checking in to see if this is something you might be able to take a look at? Thanks!

**Comment by andreujuanc** [comment]
Sorry I'm super busy now. :( Ill close this for now.

**Pull request closed without merging**